src/main/java/urlshortener/service/ShortURLBuilder.java

```diff
@@ -60,8 +64,8 @@ ShortURLBuilder temporaryRedirect() {
     return this;
   }
 
-  ShortURLBuilder treatAsSafe() {
-    this.safe = true;
+  ShortURLBuilder safe() {
```

Comment:
Maybe we should change this to `ShortURLBuilder notSafe()` or `ShortURLBuilder safe(boolean isSafe)`. I would prefer the second option because right now, if I read .safe() I would think that the URL is safe when in reality it is not.